addons/docs/src/blocks/mdx.tsx

```diff
@@ -93,6 +94,29 @@ export const AnchorMdx: FC<AnchorMdxProps> = props => {
     if (href.startsWith('#')) {
       return <AnchorInPage hash={href}>{children}</AnchorInPage>;
     }
+    // allow href="?path=/" and href="/?path=/"
+    const storyPath = href.split('?path=/');
```

Comment:
As discussed, IMO if there isn't any specific that requires the use of `?path=/` I would not support it.

 `story=/addons-docs-docs-only--page` is awesome!